File: desafio-02/talita048/python/desafio02.py
numeros = list(range(3, 10001, 2))

print (2)
for numero in numeros:
    if numero in numeros:
        print (numero)
        for multiplo in range(numero * 2, 10001, numero):
            if multiplo in numeros:
                numeros.remove(multiplo)

File: desafio-02/ricstn/javascript/desafio02.js
const numeros = [];
for (let i = 1; i <= 10000; i++) {
    let contador = 0;
    for (let j = i; j > 0; j--) {
        if (i % j === 0) {
            contador += 1;
        }
        if (contador > 2) {
            break;
        }
    }
    if (contador === 2) {
        numeros.push(i);
    }
}
console.log(numeros);
